returnn/frontend/ckpt.py

```diff
@@ -0,0 +1,31 @@
+"""
```

Comment:
The file name is misleading. `ckpt` is usually for (model) checkpoints.

In TF, there is a file `custom_gradient.py`, where you find functions like `custom_gradient` and `recompute_grad`. We actually already have `returnn/frontend/gradient.py` where you also find some similar related functions. I think the best fit would be there.